frontend/src/components/HomePage/index.jsx

```diff
@@ -4,15 +4,38 @@ import './HomePage.css';
 import Card from './Card';
 import Banner from './Banner';
 import Filters from './Filters';
+import { fetchBillsBlocks } from '../../requests';
+
+/** Simple enum to show request status */
+const RequestStatus = {
+  NONE: 'none',
+  FETCHING: 'fetching',
+  DONE: 'done',
+};
+
+/** Only search for sunrise campaign bills
+ *
+ * Gets bills for all campaigns, even if user has filtered out some specific campaigns
+ */
+const addCampaignBillsToSearchObj = (campaignMappings, searchObj) => {
+  const copy = { ...searchObj };
```

Comment:
this is redundant, since spread should already shallow copy. you can just spread the object in line 25:
```
return { ...searchObj, basePrintNo. Array.from(billIds) }
```